code/app/build.gradle

```diff
@@ -1,5 +1,7 @@
-apply plugin: 'com.android.application'
-apply plugin: "com.diffplug.spotless"
+plugins {
+    id 'com.android.application'
+    id 'com.diffplug.spotless'
+}
 
 spotless {
```

Comment:
We can take this in a separate PR, but can we define these spotless settings only once for all sources (source code, test app, tests)? 
cc @kevinlind

Reply:
One possible option is to create a common Gradle file for the Spotless configuration and apply it to the other build.gradle files. Here is an example of Core's Checkstyle configuration https://github.com/adobe/aepsdk-core-android/blob/dev-v2.0.0/code/checkstyle.gradle. This file is then included using `apply from: '../checkstyle.gradle'`.
For some of our extensions, such as Identity and Bridge, which have Kotlin test apps, the Spotless configurations are different so a common Gradle file would not help much.